### PR TITLE
Namespace filter in the Nodes tab #374

### PR DESCRIPTION
**Pull request by m2lusen**
closes #

## 📑 Description
Added the function getColumnFilterProps, based on getColumnSearchProps. getColumnFilterProps instead of a search bar creates checkboxes for each of the values (namespace in this case). Using handleSearch the selected values are then filtered, to display only the selected values.

getColumnFilterProps was added to namespace

## ✅ Checks
- [ y] I have updated the documentation as required
- [ y] I have performed a self-review of my code

## ℹ Additional context
NA

### CODE REVIEW

File: kubectl
Comment (petar-cvit) [inline kubectl:1]: Hey @m2lusen, can you remove this file?

File: cyclops-ui/src/components/pages/NodeDetails.tsx [hunk 1]
```diff
@@ -190,6 +191,97 @@ const NodeDetails = () => {
...
+      );
+
+      return (
+        <div
+          style={{
+            padding: 8,
+            maxHeight: 200,
+            overflowY: "auto",
+            margin: "8px 0",
+            border: "1px solid #ccc",
```
Comment (petar-cvit) [inline cyclops-ui/src/components/pages/NodeDetails.tsx:215]: Can we remove the border?

File: cyclops-ui/src/components/pages/NodeDetails.tsx [hunk 2]
```diff
@@ -190,6 +191,97 @@ const NodeDetails = () => {
...
+              }}
+            >
+              Close
+            </Button>
+          </Space>
+        </div>
+      );
+    },
+    filterIcon: (filtered: boolean) => (
+      <SearchOutlined style={{ color: filtered ? "#1677ff" : undefined }} />
```
Comment (petar-cvit) [inline cyclops-ui/src/components/pages/NodeDetails.tsx:271]: Can you set `#ff8803` here?  Its Cyclops orange 😄 

I see that we left the same blue on line 169, so could you also update that?

**Review by petar-cvit — CHANGES_REQUESTED** [review]
Hey @m2lusen, thanks a lot for the PR. I left a couple of comments, but looks great 🧡

File: kubectl
Comment (m2lusen) [reply kubectl:1]: Just removed it now

File: cyclops-ui/src/components/pages/NodeDetails.tsx (hunk 2 above)
Comment (m2lusen) [reply cyclops-ui/src/components/pages/NodeDetails.tsx]: Ive changed it back to orange... honestly have no idea why i changed it to that blue color, i must of used it on a different file and copied it or something.

**Review by petar-cvit — APPROVED** [review]
Thanks @m2lusen, looks great 🧡

**Pull request merged**